daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -422,7 +425,7 @@ class Engine(config: EngineConfig = EngineConfig.Stable) {
     }
   }
 
-  def enableStackTraces(enable: Boolean) = {
+  private[this] def enableStackTraces(enable: Boolean) = {
     compiledPackages.stackTraceMode =
       if (enable) speedy.Compiler.FullStackTrace else speedy.Compiler.NoStackTrace
```

Comment:
Maybe pass this as constructor parameter to `ConcurrentCompiledPackages` in line 50?